tests/rptest/tests/schema_registry_test.py

```diff
@@ -1078,3 +1112,971 @@ def test_serde_client(self):
                 assert schema.json().get("schemaType") is None
             else:
                 assert schema.json()["schemaType"] == protocols[i].name
+
+
+class SchemaRegistryBasicAuthTest(SchemaRegistryEndpoints):
```

Comment:
can you split out the changes to kwargs plumbing as a commit before this?

Reply:
Yes. I'm also cleaning this PR up now.